scripts/build_locally.sh

```diff
@@ -18,10 +18,8 @@ echo "Building the RPM ..."
 rpmbuild -ba packaging/convert2rhel.spec --define "debug_package %{nil}"
 echo "RPM was built successfully"
 echo "Cleaning up the target directory..."
-mkdir -p .rpms
-mkdir -p .srpms
-rm -frv .rpms/*
-rm -frv .srpms/*
-mv -vf ~/rpmbuild/RPMS/noarch/* .rpms/
-mv -vf ~/rpmbuild/SRPMS/* .srpms/
+mkdir -p /.rpms
+mkdir -p /.srpms
```

Comment:
We want to really create the folders in `/`? Since the script is for locally building, maybe we want to avoid that?

Reply:
Ahh right I was just thinking about containers.. good point. I just needed a way to have read-only project root but read-write .srpm and .rpm directory... hmm

Reply:
@r0x0d done! Should be fixed now